src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
[FUNCTION NAMING] - The `hide` method should only set the `hidden` property to `true`, but currently it's not clear if it's meant to toggle the value. To adhere to the checklist, ensure that this method only hides the animal, without the possibility of un-hiding it.